docs/Migrating.md

```diff
@@ -33,6 +33,11 @@ double-check that the versions are in the same. The versions can be found in
 - The signature of `Agent.Heuristic()` was changed to take a `float[]` as a
   parameter, instead of returning the array. This was done to prevent a common
   source of error where users would return arrays of the wrong size.
+- The SideChannel API has changed:
```

Comment:
We need to mention the user-facing changes of going from StatsSideChannel to Academy.Instance.StatsRecorder

Reply:
And that StatsSideChannel is now internal